test/evergreen/run_format_configs.sh

```diff
@@ -10,7 +10,7 @@ set -e
 # Switch to the Git repo toplevel directory
 cd $(git rev-parse --show-toplevel)
 # Walk into the test/format directory
-cd cmake_build/test/format
+cd build/test/format
```

Comment:
this looks like a testing change that accidentally got included?

Reply:
Yes this is true, can change it back.